omnibus/package-scripts/updater-deb/postinst

```diff
@@ -107,9 +106,15 @@ if [ -d "$CONFIG_DIR/runtime-security.d" ]; then
     chown -R root:root ${CONFIG_DIR}/runtime-security.d || true
 fi
 
+echo "api_key: 00000000000000000000000000000000" > $CONFIG_DIR/datadog.yaml # todo remove this once in install script
```

Comment:
I don't know how it should run but we might check if the file exists and/or if `api_key` is set and update it accordingly, instead of erasing it?